tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "isolatedModules": true,
```

Comment:
Out of curiosity, why is this needed?

Reply:
I've copied this from some blog post. Will remove it now.